app/services/exports/trainee_search_data.rb

```diff
@@ -121,6 +121,11 @@ def course_summary(trainee, course)
       end
     end
 
+    def funding_manager(trainee)
```

Comment:
The variable here isn't necessary. Without the `@` its scope is internal to the method so this wouldn't memoise it. We probably don't want to memorise this anyway as we'd end up with a the `FundingManager` stuck on the same trainee.
```suggestion
    def funding_manager(trainee)
      FundingManager.new(trainee)
    end
```